src/components/asthma/components/AsthmaAirQualities/AsthmaAirQualities.tsx

```diff
@@ -0,0 +1,110 @@
+
+export default function (props: AsthmaAirQualitiesProps) {
+    const [loading, setLoading] = useState<boolean>(true);
+    let [homeAirQuality, setHomeAirQuality] = useState<AsthmaAirQuality>();
+    let [workAirQuality, setWorkAirQuality] = useState<AsthmaAirQuality>();
+
+    useInitializeView(() => {
+        setLoading(true);
+
+        if (props.previewState === 'loading') {
+            return;
+        }
+        if (props.previewState) {
+            setHomeAirQuality(previewData[props.previewState].homeAirQuality);
+            setWorkAirQuality(previewData[props.previewState].workAirQuality);
+            setLoading(false);
+            return;
+        }
+
+        asthmaDataService.loadParticipant().then(participant => {
+            let loadAirQualities = props.date ?
+                asthmaDataService.loadAirQualitiesForDate(props.date) :
+                asthmaDataService.loadAirQualitiesForControlStatus(participant.getHomeAirQualityZipCode(), participant.getWorkAirQualityZipCode());
+
+            loadAirQualities.then(airQualities => {
+                setHomeAirQuality(airQualities.find(aq => aq.type === 'home'));
+                setWorkAirQuality(airQualities.find(aq => aq.type === 'work'));
+                setLoading(false);
+            });
+        });
+    }, [], [props.previewState]);
+
+    const onSetup = (): void => {
+        if (props.previewState || !props.editZipCodesSurveyName) return;
+        MyDataHelps.startSurvey(props.editZipCodesSurveyName);
+    }
+
+    const getSetupComponent = (label: string) => {
+        return <UnstyledButton onClick={() => onSetup()}>
+            <div className="mdhui-asthma-air-qualities-setup">
+                <div className="mdhui-asthma-air-qualities-setup-button">{language('asthma-air-qualities-setup-button-text')}</div>
+                <div className="mdhui-asthma-air-qualities-setup-label">{label}</div>
+            </div>
+        </UnstyledButton>;
+    };
+
+    const onClick = (airQuality: AsthmaAirQuality): void => {
+        if (props.previewState) return;
+        if (airQuality.status === 'not-configured' && props.editZipCodesSurveyName) {
+            MyDataHelps.startSurvey(props.editZipCodesSurveyName);
+        } else {
+            MyDataHelps.openApplication(props.airQualityUrl, {modal: true});
+        }
+    };
+
+    return <div className="mdhui-asthma-air-qualities" ref={props.innerRef}>
+        <div className="mdhui-asthma-air-qualities-header">
+            <div className="mdhui-asthma-air-qualities-title">{language('asthma-air-qualities-title')}</div>
+            {props.editZipCodesSurveyName &&
+                <UnstyledButton className="mdhui-asthma-air-qualities-settings" onClick={() => onSetup()}>
+                    <FontAwesomeIcon icon={faGear}/>
+                </UnstyledButton>
+            }
+        </div>
+        {loading && <LoadingIndicator/>}
+        {!loading &&
+            <div className="mdhui-asthma-air-qualities-data">
```

Comment:
Minor suggestion: It might be useful to actually create a `GridLayout` or something to that effect as a generic Presentational Component. It recently came up in other conversations that someone wanted to put two buttons on the same line; same problem you're having here with laying out `SingleDataPoints`.  The css class you added here and for Biometrics presentation are identical, which also sort of helps imply that this this could be a unified presentation option.

Reply:
When I first started working on this, I actually had a prototype for a `ColumnLayout` component that allowed for a configurable number of columns.  I punted on that due to time constraints, but I agree that a reusable multi-column layout component would be nice to have for use here and other places.  I'll probably work on something like that in a follow up PR.